src/test/tests.ts

```diff
@@ -234,6 +241,31 @@ describe('SubscriptionManager', function() {
     });
   });
 
+  it('can use an async filter function', function(done) {
```

Comment:
yeah i've seen it, but it means the test is actually making sure promises work, 
and not that things are running in parallel..
name is abit confusing.. 🗡